apps/site/config/test.exs

```diff
@@ -3,7 +3,7 @@ use Mix.Config
 # We don't run a server during test. If one is required,
 # you can enable the server option below.
 config :site, SiteWeb.Endpoint,
-  http: [port: 4002],
+  http: [port: System.get_env("PORT") || 4002],
```

Comment:
Does this work? `System.get_env("PORT")` is going to be a string, not an integer.

Reply:
It does work for the sake of running the app, despite being a string!